pkg/ebpf/ingress_node_firewall_statistics.go

```diff
@@ -5,23 +5,25 @@ import (
 	"time"
 )
 
-const maxRules = 100
+const maxRules = 100 // Max number of rules per sourceCIDRs
 
-func ingressNodeFwStats(objs bpfObjects) error {
-	// Read loop reporting the total amount of times the kernel
-	// function was entered, once per second.
-	ticker := time.NewTicker(1 * time.Second)
-	defer ticker.Stop()
-	log.Printf("Waiting for statistics update")
-	for range ticker.C {
-		var ruleStats bpfRuleStatisticsSt
-		var totalStats []bpfRuleStatisticsSt
-		for rule := 0; rule < maxRules; rule++ {
-			if err := objs.IngressNodeFirewallStatisticsMap.Lookup(&rule, &ruleStats); err != nil {
-				return err
+// ingressNodeFwStats Collect Ingress node firewall per rule statistics
+func (infc *IngNodeFwController) ingressNodeFwStats() error {
+	objs := infc.objs
+	go func() {
```

Comment:
nit: This doesn't gracefully finish when we exit.

Reply:
yeah this is more or less just skelton for the stats part